Bottom: e62431e2c311c173f6a832272e1c9e4faaaf8ddb
Top:    b7fcc6fd63defb30a6ccd179b98b8eb84045b47c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-10 17:12:11 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 3756955..33b7650 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1190,6 +1190,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1212,6 +1213,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
